lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -274,6 +274,12 @@ class AwsCompileEventBridgeEvents {
     // It suggests that the object already exists and is being imported
     if (_.isObject(eventBusName)) {
       eventBusExists = true;
+      if (eventBusName['Fn::GetAtt'] && eventBusName['Fn::GetAtt'][1] !== 'Name') {
```

Comment:
I think it'll be nice to fine tune config schema, so it supports just expected format.

While it won't provide as nice message, it'll clearly indicate which property is not supported.

What do you think?

Reply:
Good suggestion - I'll adjust it